test/integration_test/clusterdomain_test.go

```diff
@@ -156,11 +163,11 @@ func testClusterDomainsFailover(
 	err = setRemoteConfig(remoteFilePath)
 	require.NoError(t, err, "Error setting remote config")
 
-	// Increase the replicas on cluster 1
+	// Set scale factor to it's orignal values on cluster 1
 	scaleFactor, err = schedulerDriver.GetScaleFactorMap(preMigrationCtx)
 	require.NoError(t, err, "Unexpected error on GetScaleFactorMap")
 	for k := range scaleFactor {
```

Comment:
Don't need this loop, just pass in oldScaleFactor to scale up

Reply:
Fixed.